nodejs/hardhat.config.js

```diff
@@ -0,0 +1,16 @@
+/** @type import('hardhat/config').HardhatUserConfig */
+require("@nomicfoundation/hardhat-toolbox");
+require('dotenv').config({path: '.env'});
+const arkhiaJsonRpcRelayTestnet  = `${process.env.ARKHIA_TESTNET_URL}/${process.env.ARKHIA_API_KEY}`;
+const operatorPrivateKey = process.env.OPERATOR_PRIVATE_KEY;
```

Comment:
This can also be in the urlHandler, perhaps the name can change a bit.

Reply:
This is done